sbol_utilities/helper_functions.py

```diff
@@ -357,3 +357,12 @@ def collector(obj: sbol3.Identified):
     with cached_references(doc):
         doc.traverse(collector)
     return outgoing
+
+def find_feature(doc: sbol3.Component,f:sbol3.Feature) -> sbol3.Feature:
+    """
+    find_feature function used for  finding a specific feature you specify and check
+    if this feature one of your sbol3 document features or not.
```

Comment:
It is not a document, but a component. You should change the variable name as well.